docs/guide/installation.md

```diff
@@ -64,7 +64,7 @@ If you are using `enableStrictParsing` URL manager option, add the following to
 ### Extra configuration for logging and profiling
 
 Logging and profiling are simple but powerful tools that may help you to understand the execution flow of both the
-framework and the application. These tools are useful for development and production environments alike.
+framework and the application. These tools are useful for development and production environments NOT alike.
```

Comment:
@samdark in my opinion they could be useful for both environments, but useful not alike.
Logging and profiling are less destructive in development environment, but can hurt performance in production environment if will be used incorrectly.
But anyway I already revert change.